marcdown_project/frontend/src/components/common/Nav.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+
+import {
+  NavLink,
+} from "react-router-dom";
+import { HashRouter } from "react-router-dom/cjs/react-router-dom.min";
+
+import NavItem from "./NavItem"
+
+
+class Nav extends Component {
+  render() {
+    return (
+      <HashRouter>
+        <nav>
+          <NavItem url="/" text="" exact={true} >
+            <img src="/static/images/logo.png" alt="" />
+          </NavItem>
+          <ul>
+            <li>
+              <NavItem url="/" exact={true}>Home</NavItem>
+            </li>
+            <li>
+              <NavItem url="/editor">Editor</NavItem>
```

Comment:
How is the editor linked in the navigation bar ?

Reply:
Fixed it, replaced by /note/:id